storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/CommonRestResponse.java

```diff
@@ -50,7 +50,7 @@ public int statusCode() {
     /**
      * @return An HTTP Etag for the blob at the time of the request.
      */
-    public String eTag() {
+    public String getEtag() {
```

Comment:
This actually intentionally does not have "get" as a prefix. All of the getters generated by autorest follow this patern of simply having the field as the method name, and we were very conscious about following that, so I think this should be reverted.

Reply:
Ok, I will suppress the spotbugs.